bundles/Billing/Service/BillingService.php

```diff
@@ -30,27 +31,30 @@ class BillingService extends JiraService
     private $expenseRepository;
     private $invoiceRepository;
     private $boundReceiverAccount;
+    private $cache;
 
     /**
      * Constructor.
      *
-     * @param \Doctrine\ORM\EntityManagerInterface $entityManager
      * @param $jiraUrl
      * @param $tokenStorage
      * @param $customerKey
      * @param $pemPath
+     * @param CacheProvider                         $cache
+     * @param \Doctrine\ORM\EntityManagerInterface  $entityManager
      * @param \Billing\Repository\WorklogRepository $worklogRepository
      * @param \Billing\Repository\ExpenseRepository $expenseRepository
      * @param \Billing\Repository\InvoiceRepository $invoiceRepository
      * @param $boundReceiverAccount
      * @param $boundCustomFieldMappings
      */
     public function __construct(
-        EntityManagerInterface $entityManager,
         $jiraUrl,
         $tokenStorage,
         $customerKey,
         $pemPath,
+        CacheProvider $cache,
+        EntityManagerInterface $entityManager,
```

Comment:
Is this entity manager actually used? No instance variable is set in the constructor.

Reply:
Øhh... It is used multiple places :)
The reason it is in included in the PR, is because I change the order of injections in the constructor.